docker-compose.yml

```diff
@@ -17,6 +17,15 @@ services:
       - PREFIX=
       - SUGGESTIONS_CHANNEL=
       - PYTHONUNBUFFERED=1
+
+      # level of logging displayed in the console (20 means INFO and above):
+      # 10 DEBUG
+      # 20 INFO
+      # 30 WARNING
+      # 40 ERROR
+      # 50 CRITICAL
+      - LOGGING_LEVEL=
```

Comment:
what would be the default for this

Reply:
If left empty, the code would probably error out due to the way the reading of env values is implemented.

I would recommend info level for almost all cases. Debug will communicate most discord API interactions, and there are a lot of API messages that you don't need when running prod or even feature tests.

Reply:
If you default it to info I'll mark it as approved